.github/ISSUE_TEMPLATE/Style.yml

```diff
@@ -0,0 +1,27 @@
+name: 👯‍♂️ Style Changing Request
+description: Suggest a style designs
+title: '[style]'
+labels: ["enhancement"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this template!
+
+  - type: textarea
+    id: style-idea
+    attributes:
+      label: What's the style idea?
+      placeholder: Add descriptions
+      value: 'We need to improve'
+      validations:
```

Comment:
The additional 2 spaces before `validations` causes this error:

> body[1]: validations is not a permitted attribute. [Learn more about this error.](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#bodyi-x-is-not-a-permitted-attribute)

Remove those two spaces may fix this issue.